torchgeo/datasets/oscd.py

```diff
@@ -290,10 +307,15 @@ def plot(
         """
         ncols = 2
 
-        rgb_inds = [3, 2, 1] if self.bands == "all" else [0, 1, 2]
+        try:
+            rgb_indices = [self.bands.index(band) for band in self.rgb_bands]
+        except ValueError as e:
+            raise ValueError(
+                "RGB bands must be present to use `plot` with Sentinel-2 imagery."
```

Comment:
Never heard of `raise from` before, neat!

Btw, you'll need to add a simple test to cover this line. Should be plenty of examples in other datasets.